### PR TITLE
Feb newsletter draft

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

File: docs/blog/newsletter-february-2020.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
+.. post:: February 03, 2020
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2020
+#########################################
+
+Happy 2020, documentarians! It's Beth and the newsletter crew here, with the latest and greatest from the Write the Docs community.
+
+Two bits and pieces for you on the news front today. In December we annouced dates for the Prague conference, which will be 13-15 September - and in a brand shiny new venue, the Hotel Olšanka! `Read all about it here </conf/prague/2020/news/init/>`__. As for across the pond, the speakers for Portland speakers will be announced soon, so keep your eyes peeled for those. The conference is 3-5 May, and `tickets are on sale right now </conf/portland/2020/tickets/>`__.
```
Comment (jrondeau-stripe) [inline docs/blog/newsletter-february-2020.rst]: Maybe I'm being unduly fussy here, but the first sentence here reads a bit too colloquially (I first read "two bits" and then had to reorient myself to the rest of the sentence, and still got tripped up on "news front" ..)
Maybe trim way down, to something like "Two news items today"?
And rm "As for" in the second sentence, so it starts with "Across the pond ..."

File: docs/blog/newsletter-february-2020.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+So with that, let's see what's been cooking on Slack recently!
+
+-----------------------------
+#learn-tech-writing book club
+-----------------------------
+
+There's been a buzz of activity all through January in the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ channel, where folks have been discussing *The Product is Docs* chapter-by-chapter. As those involved have kindly started an effort to write up all the fantastic conversations on the topic, we're not going to write it up in the newsletter, but we wanted to point you towards what's been written up so far:
```
Comment (jrondeau-stripe) [inline docs/blog/newsletter-february-2020.rst]: lol the hyphenated construction which was ALSO discussed on Slack recently (or maybe it was on work Slack, I can't keep track anymore ...) -- anyway, pretty sure it's "chapter by chapter" w/o hyphens, although that's a nit that you can ignore if you like.
That's also three instances of "write up" and variants in a single sentence. Maybe something more like"
"Participants have started a writeup of all the fantastic conversations the discussion produced, so we won't duplicate their efforts here. But here's a list of what they've produced so far:"

File: docs/blog/newsletter-february-2020.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- `Technical editing <https://github.com/writethedocs/www/blob/bookClubWriteUps/docs/book-club/splunk-product-docs/13-technical-editing.rst>`__
+
+If you'd like to help out with write-ups, drop into the channel!  As a bonus, they're looking into what to discuss next, so if you have a book you think would be great for group discussions, go ahead and make the suggestion. 
+
+---------------------------------------------
+Supporting documentation, documenting support
+---------------------------------------------
+
+A recent question about how to balance docs work and support content produced some interesting responses. The original post assumed two distinct teams, one for documentation and one for content produced by a support team, with each team producing distinct content. This is evidently the situation in many companies. For example, one poster described a workflow where answers to frequently-asked support questions are added to the documentation, but if a potential docs page seems particularly complicated, or addresses an edge case, the writer asks support to document it. This story describes distinct teams -- Docs and Support -- and distinct types of content.
```
Comment (jrondeau-stripe) [inline docs/blog/newsletter-february-2020.rst]: hah I get to edit my own stuff. That first sentence is really lame. How about "... produced a range of unexpected responses." instead?
Also, too many "distinct"s but I haven't got it in me to fix. (Two in the last sentence are deliberate and useful, but the earlier one not so much :-( )

File: docs/blog/newsletter-february-2020.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+But many replies to the original post were from people who work on support teams, not on dedicated docs teams. And they write the docs. In their situations, there isn't a clear distinction between roles, nor between types of content. More than one reply described scenarios in which all docs are produced by support. 
+
+Still other replies came from technical writers whose role emerged from support. Some writers report to a support manager, and some people in a dedicated writer role work from support tickets, to make sure the docs address known customer issues. Some folks work on both product docs and separate knowledgebase articles.
+
+In short, there are myriad ways to organize the relationship between docs teams and support teams, and between the different (or not-so-different) types of content. What everyone seemed to agree on, though, was the need for close and well-coordinated communication across teams and deliverables.
+
+--------------------------------------
+How to set priorities in documentation
```
Comment (jrondeau-stripe) [inline docs/blog/newsletter-february-2020.rst]: "How to set priorities on a documentation team" ? Seems like prioritizing isn't a team-only issue, so title here could be clearer ...

File: docs/blog/newsletter-february-2020.rst [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+So you're no longer the only documentarian in your company. Hooray! Working in a team allows you to tackle bigger and more complicated projects. But how do you decide what to focus on, especially for new team members? This month, our community discussed what helps them set priorities:
+
+* Discuss the project's roadmap with project/product managers, and ask them (and other stakeholders) about immediate documentation needs. For your new writers, ask about topics that could use a little extra attention - an opportunity for them to specialize in something.
+* Encourage new writers to sit in during backlog grooming and project demos. Product prioritization can inform your own priorities.
+* Coordinate with engineering teams by attending their standup and planning meetings. You're working towards a common goal, so it helps to know how everyone's contributions fit together.
+* Try an issue tracking tool (eg JIRA or GitHub Issues). Some feel it helps keep them accountable and prevents them from struggling under the workload. Similarly, for big projects, try mapping them out using a tool, to give your team and managers a high-level glance at everyone's workload.
```
Comment (jrondeau-stripe) [inline docs/blog/newsletter-february-2020.rst]: s/JIRA/Jira (how the product itself capitalizes, although it's true it's not consistent everywhere. But I believe it isn't an acronym, so might as well keep to the main UI as a reference point)

File: docs/blog/newsletter-february-2020.rst [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+- `The State of Unassisted Support 2014 <http://cdn.swcdn.net/creative/pdf/Whitepapers/SW-WP-The-State-of-Unassisted-Support-2014.pdf>`_, Technology Services Industry Association (TSIA). Includes a graph that compares the cost of self-service support with the costs of other methods.
+- `A Good User's Guide Means Fewer Support Calls and Lower Support Costs <www.jstor.org/stable/43095065>`_ by Cathy J. Spencer and Diana Kilbourn Yates. GE Information Services saved $1 million in support costs per year by providing good user documentation with software products.
+- `Currents, A Seasonal Report on Developer Trends in the Cloud: Open Source Edition <https://www.digitalocean.com/currents/october-2018>`_, Digital Ocean. For open source technologies, great documentation can reduce the barrier to entry for contributors, and it's something companies are looking for when they evaluate open source products.
+
+----------------------------------------------
+Dealing with finding inaccuracies in your docs
```
Comment (jrondeau-stripe) [inline docs/blog/newsletter-february-2020.rst]: That's ... a lot of gerunds. "When you find inaccuracies in your docs"? Or just "Inaccuracies in your docs"?

File: docs/blog/newsletter-february-2020.rst [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+Although we strive to write accurate documentation, inevitably there are times when errors are made that require addressing. This month our community talked about what to do when you realise there are inaccuracies in published articles.
+
+The question of blame is unavoidable. Should writers have their work scrutinized for small errors? Many felt problems in documentation should rather be attributed to process failure. If you have to complete a document in a limited time, the pressure of the crunch can result in not enough time to check, and thereby faulty docs. Others didn't want to frame it as a failure: just a chance to improve both the doc and processes. But even if you have a good pre-publication process, errors may arise from other factors: for example, testing the docs but within the wrong environment, meaning they don't match the end user experience.
+
+What most agreed on is that the documentation should be revised and corrected as soon as any errors are discovered. If errors are a big problem, teams can establish a process of reviews and approvals to check for accuracy. Another option suggested was be to have a QA function test the documentation.
```
Comment (jrondeau-stripe) [inline docs/blog/newsletter-february-2020.rst]: rm "be" and trim last sentence: "Another suggestion was to have QA test the documentation."

**Comment by jrondeau-stripe** [comment]
OK, tossed a bunch o' single comments in; lmk when you'd like a re-review (and I'll change my login, with apologies for any current confusion)

File: docs/blog/newsletter-february-2020.rst (hunk 1 above)
Comment (plaindocs) [reply docs/blog/newsletter-february-2020.rst]: Or just "Some bits and pieces"

File: docs/blog/newsletter-february-2020.rst [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+So with that, let's see what's been cooking on Slack recently!
+
+-----------------------------
+#learn-tech-writing book club
+-----------------------------
+
+There's been a buzz of activity all through January in the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ channel, where folks have been discussing *The Product is Docs* chapter-by-chapter. As those involved have kindly started an effort to write up all the fantastic conversations on the topic, we're not going to write it up in the newsletter, but we wanted to point you towards what's been written up so far:
+
+- `Agile and technical writing <https://github.com/writethedocs/www/blob/bookClubWriteUps/docs/book-club/splunk-product-docs/2-agile.rst>`__
+- `Documenting third-party products <https://github.com/writethedocs/www/commit/89ead30998101c1ce1038203c74e53aa50719a29>`__
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2020.rst:21]: Would be better to link to a PR than a branch here, but I guess that is up to @StarfallProjects

File: docs/blog/newsletter-february-2020.rst (hunk 3 above)
Comment (plaindocs) [inline docs/blog/newsletter-february-2020.rst]:
```suggestion
A recent question about how to balance docs work and support content produced some enthusiastic discussion. The original post assumed two separate teams, one for documentation and one for content produced by a support team, with each team producing their own content. This is evidently the situation in many companies. For example, one poster described a workflow where answers to frequently-asked support questions are added to the documentation, but if a potential docs page seems particularly complicated, or addresses an edge case, the writer asks support to document it. This story describes distinct teams -- Docs and Support -- and distinct types of content.
```

Comment (plaindocs) [reply docs/blog/newsletter-february-2020.rst]: Something like this :point_up: ? Edit as needed. Also 

> The original post assumed two separate teams, one for documentation and one for content produced by a support team

reads super odd to me. One for docs and one for support.

File: docs/blog/newsletter-february-2020.rst (hunk 4 above)
Comment (plaindocs) [reply docs/blog/newsletter-february-2020.rst]: "How to set Documentation priorities"? :-p

File: docs/blog/newsletter-february-2020.rst [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+* Encourage new writers to sit in during backlog grooming and project demos. Product prioritization can inform your own priorities.
+* Coordinate with engineering teams by attending their standup and planning meetings. You're working towards a common goal, so it helps to know how everyone's contributions fit together.
+* Try an issue tracking tool (eg JIRA or GitHub Issues). Some feel it helps keep them accountable and prevents them from struggling under the workload. Similarly, for big projects, try mapping them out using a tool, to give your team and managers a high-level glance at everyone's workload.
+* Set up a mandatory flag or field to say if an item needs to be in the release notes. This ensures that important items and fixes won't go undocumented.
+* Audit your docs, archiving or updating content that people aren't reading. Alongside PMs, determine which content is outdated or unnecessary, and then divide them among writers on the team. And if you're struggling to persuade colleagues that an audit is necessary, our next article might just help.
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2020.rst:49]: :+1:

**Review by clairealyssa — COMMENTED** [review]
I didn't really see anything that Jennifer didn't catch already. Great job, Beth! 👍

File: docs/blog/newsletter-february-2020.rst (hunk 6 above)
Comment (plaindocs) [reply docs/blog/newsletter-february-2020.rst]: Gerunds gonna keep on gerunding @Bradamant3

File: docs/blog/newsletter-february-2020.rst [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+----------------------------------------------
+Dealing with finding inaccuracies in your docs
+----------------------------------------------
+
+Although we strive to write accurate documentation, inevitably there are times when errors are made that require addressing. This month our community talked about what to do when you realise there are inaccuracies in published articles.
+
+The question of blame is unavoidable. Should writers have their work scrutinized for small errors? Many felt problems in documentation should rather be attributed to process failure. If you have to complete a document in a limited time, the pressure of the crunch can result in not enough time to check, and thereby faulty docs. Others didn't want to frame it as a failure: just a chance to improve both the doc and processes. But even if you have a good pre-publication process, errors may arise from other factors: for example, testing the docs but within the wrong environment, meaning they don't match the end user experience.
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2020.rst]:
```suggestion
The question of blame is unavoidable. Should writers have their work scrutinized for small errors? Many felt problems in documentation should rather be attributed to process failure. If you have to complete a document to a deadline you may not have enough time to check it properly, resulting in faulty docs. Others didn't want to frame it as a failure: just a chance to improve both the doc and processes. But even if you have a good pre-publication process, errors may arise from other factors: for example, testing the docs but within the wrong environment, meaning they don't match the end user experience.
```

File: docs/blog/newsletter-february-2020.rst (hunk 7 above)
Comment (plaindocs) [reply docs/blog/newsletter-february-2020.rst]: QA function?

File: docs/blog/newsletter-february-2020.rst [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+---------
+Job posts
+---------
+
+* `Lead Technical Writer <https://jobs.writethedocs.org/job/171/lead-technical-writer-software-documentation/>`__
+   Deskpro, London
+* `Technical Writer <https://jobs.writethedocs.org/job/175/technical-writer/>`__
+   Wowza Media Systems, remote
+
+*To apply for these jobs and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2020.rst]: This "and more" rubs me the wrong way. Given that you can click on the individual jobs above how about 

> Find more jobs on ....

**Review by plaindocs — CHANGES_REQUESTED** [review]
Stellar stuff. Nits requested / suggested below.

File: docs/blog/newsletter-february-2020.rst (hunk 3 above)
Comment (plaindocs) [reply docs/blog/newsletter-february-2020.rst]: A rare privilege?

File: docs/blog/newsletter-february-2020.rst (hunk 1 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: I was clearly feeling a bit too flowery yesterday ;) thanks, fixed!

File: docs/blog/newsletter-february-2020.rst (hunk 2 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: Your version is far better, added

File: docs/blog/newsletter-february-2020.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: fixed the intro, changed up some instances of 'distinct', added Sam's suggestion

File: docs/blog/newsletter-february-2020.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: fixed!

File: docs/blog/newsletter-february-2020.rst (hunk 5 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: thanks for spotting, fixed!

File: docs/blog/newsletter-february-2020.rst (hunk 6 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: fixed!

File: docs/blog/newsletter-february-2020.rst (hunk 7 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: ty, fixed

File: docs/blog/newsletter-february-2020.rst (hunk 8 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst:21]: That's what was in the spreadsheet, so I agree but... I'm too lazy to dig around to get the correct link

File: docs/blog/newsletter-february-2020.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: eep, yes. fixed

File: docs/blog/newsletter-february-2020.rst (hunk 10 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: ty, added

File: docs/blog/newsletter-february-2020.rst (hunk 11 above)
Comment (bethaitman) [reply docs/blog/newsletter-february-2020.rst]: haha this has been in the template forever. I agree, have changed.

**Review by plaindocs — APPROVED** [review]

**Pull request merged**